config/settings/prod.py

```
@@ -1,8 +1,19 @@
 from .base import *
-ALLOWED_HOSTS = ["127.0.0.1", "localhost"]
+import os
+
+ALLOWED_HOSTS = ["127.0.0.1", "localhost", "api.blog.dailyfunding.cloud"]
 
 DEBUG = False
 
-INSTALLED_APPS += [
-    #
-]
+INSTALLED_APPS += []
```

Ruff (0.15.5)

[error] `INSTALLED_APPS` may be undefined, or defined from star imports

(F405)